code/game/turfs/turf.dm

```diff
@@ -79,12 +79,14 @@
 		spawn( 0 )
 			src.Entered(AM)
 			return
-	turfs |= src
```

Comment:
No I mean byondwise, you cannot ever remove a turf, any new turf simply replaces the old one and all references to it get replaced.

That's why procs running after a turf gets replaced will runtime trying to read nonexistent vars